### ISSUE TITLE
A strange warning: com.lenis0012.bukkit.loginsecurity.session.AuthAction

### ISSUE THREAD

**Issue by swordin** [lenis0012/LoginSecurity-2#108]
I got the following:

[21:07:09 WARN]: Exception in thread "pool-5-thread-67" 
[21:07:09 WARN]: java.lang.NullPointerException
[21:07:09 WARN]:        at com.lenis0012.bukkit.loginsecurity.session.AuthAction.rehabPlayer(AuthAction.java:58)
[21:07:09 WARN]:        at com.lenis0012.bukkit.loginsecurity.session.action.RegisterAction.run(RegisterAction.java:33)
[21:07:09 WARN]:        at com.lenis0012.bukkit.loginsecurity.session.PlayerSession.performAction(PlayerSession.java:168)
[21:07:09 WARN]:        at com.lenis0012.bukkit.loginsecurity.session.PlayerSession$2.run(PlayerSession.java:142)

**Comment by lenis0012** [comment]
@swordin can you give more context?
Did you recently restart or reload?
perphaps the player quit soon after logging in?

It looks like the player could not be found after a login session completed.

**Comment by swordin** [comment]
Yes, you're right. It was after the playes quit soon after logging in:

[21:06:09] [Server thread/INFO]: angel888 (95.32.7.0) enters
[… … …]        … … … 
[21:07:09] [Server thread/INFO]: [Player] angel888 leaves.
[21:07:09] [pool-5-thread-67/WARN]: Exception in thread "pool-5-thread-67"
[21:07:09] [pool-5-thread-67/WARN]: java.lang.NullPointerException

Let's close this problem.


By the way!.. Something on another topic, sorry )

You know the biggest disadvantage is that the player's login message comes into the chat BEFORE the player is authorized. (I already talked about this in another issue.) The new player has not yet entered a password (maybe it was hacker?), but everyone thinks that he has already entered to the server and begin to talk with him.  

Next day the other players ask him «Hey, why did you not answer when you entered?». And he answers: «Wow, I did not have a computer at all yesterday!». So the players have a lack of trust in the server security.

I am convinced that messages about the entry of an unauthorized player should come to the console only. And when he was logged in - then in the chat.

So we all are waiting new version of your plugin! )
Thank you!
